common/lsp/lsp-protocol.yaml

```diff
@@ -202,3 +202,17 @@ DocumentLinkParams:
 DocumentLink:
   range: Range
   target?: string  # DocumentUri
+
+# -- textDocument/prepareRename   (TODO(jbylicki): check that requires project + active symbol table #1189)
+PrepareRenameParams:
+  <: TextDocumentPositionParams
+
+# Response: Range[]
```

Comment:
Isn't the response just a `Range`, not `Range[]`  ( https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#textDocument_prepareRename ) ?

Reply:
Yes it is! I have at one point misread the spec and implemented it for `Range[]`. It was fixed within the code itself, but I forgot it here. Changed it now